src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -259,6 +273,17 @@ private static boolean shouldBeAloneOnLine(RightCurlyOption bracePolicy, Details
     }
 
     /**
+     * Checks that a right curly should be only with semi on a line.
+     * @param bracePolicy option for placing the right curly brace
+     * @param details Details for validation
+     * @return true if after right curly should be SEMI.
+     */
+    private static boolean shouldBeAloneWithSemi(RightCurlyOption bracePolicy, Details details) {
```

Comment:
@MEZk , it is ok, at least it follow naming in class , all methods with "should...." check an user mode property.